frontend/components/AddSite/TemplateSiteList/templateSite.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 
+import { getSafeRepoName } from '../../../util';
+
 const propTypes = {
   name: React.PropTypes.string.isRequired,
-  owner: React.PropTypes.string.isRequired,
-  branch: React.PropTypes.string.isRequired,
-  repo: React.PropTypes.string.isRequired,
```

Comment:
Just checked. Those used to be used, but fell out of usage in commit 9b0b2e6554a61f14073dd51c34fa4f115648e154